src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -1,12 +1,13 @@
 <div class="col-md-12 col-xs-12">
   <div class="alert alert-danger" role="alert" ng-messages="$ctrl.signUpErrors" ng-if="$ctrl.hasError">
-    <div ng-message="400" translate>An error occurred during sign up.<br />Please Check your Email Address and Password and try again.</div>
-    <div ng-message="403" translate>An error occurred during sign up.<br />This is most likely caused by the use of a reserved domain name in your email address. You might try to
+    <div ng-message="invalid_password_message">{{$ctrl.signUpErrors.invalid_password_message}}</div>
+    <div ng-message="400" translate>An error occurred during sign up. <br />Please Check your Email Address and Password and try again.</div>
+    <div ng-message="403" translate>An error occurred during sign up. <br />This is most likely caused by the use of a reserved domain name in your email address. You might try to
```

Comment:
Is there a reason for these whitespace changes?

Reply:
When the error was rendered, it appeared with no whitespace between the period and the next word. 
```An error occurred during sign up.Please Check your Email Address and Password and try again.```
```This is most likely caused by the use of a reserved domain name in your email address.You might try to...`